### PR TITLE
Add v11 for content-encoding:br Safari support

### PR DESCRIPTION
**Pull request by whitep4nth3r**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Updated http -> headers -> content-encoding to reflect that content-encoding: br has been supported in Safari since version 11.

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

See: https://caniuse.com/?search=content-encoding%20br

- [x] Data: if you tested something, describe how you tested with details like browser and version
I have verified this behaviour by checking in the Sarafi network tab on multiple sites serving content with the http header content-encoding: br.

- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)

N/A

- [x] Link to related issues or pull requests, if any

N/A

### CODE REVIEW

**Comment by gsnedders** [comment]
While mostly historic at this point, we probably want to note that support is only when running Safari on macOS 10.13 High Sierra or later.

File: http/headers/content-encoding.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "11"
```
Comment (ddbeck) [inline http/headers/content-encoding.json:81]: Proposing some note text based on https://github.com/mdn/browser-compat-data/pull/10330#issuecomment-839978678

```suggestion
                "version_added": "11",
                "notes": "Unsupported before macOS 10.13 High Sierra."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @whitep4nth3r. Also, I see that it's your first on the project—welcome!

One suggestion in a line comment, but this is nearly ready to go. Thank you again!

**Comment by whitep4nth3r** [comment]
Sorry for the delay! Thank you for the suggestion!

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**